.travis.yml

```diff
@@ -20,7 +26,7 @@ r_github_packages:
   - USGS-R/rloadest@v0.4.2
 
 after_success:
-  - Rscript -e 'covr::codecov(); covr::coveralls()'
+  - Rscript -e 'library(covr);covr::codecov(); covr::coveralls()'
```

Comment:
Is `library(covr)` really needed? Unless it's written oddly, `covr::` should be sufficient, and both `codecov()` and `coveralls()` were running before this PR.